components/autosize_textarea.tsx

```diff
@@ -131,6 +162,8 @@ export default class AutosizeTextarea extends React.Component {
         const placeholderAriaLabel = placeholder.toLowerCase();
         if (!this.props.value && !this.props.defaultValue) {
             textareaPlaceholder = (
+
+                // @ts-ignore for now
```

Comment:
I've added that to pass the lint error, because the some of the props spread is not compatible with div(?)

```ts
 onChange?: ((e: React.ChangeEvent<HTMLTextAreaElement>) => void) | undefined;
 onHeightChange?: ((height: number, maxHeight: number) => void) | undefined;
 children?: React.ReactNode;
```